x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/action/PutTrainedModelActionRequestTests.java

```diff
@@ -22,7 +22,7 @@ protected Request createTestInstance() {
         String modelId = randomAlphaOfLength(10);
         return new Request(
             TrainedModelConfigTests.createTestInstance(modelId, false)
-                .setParsedDefinition(TrainedModelDefinitionTests.createRandomBuilder())
+                .setParsedDefinition(TrainedModelDefinitionTests.createSmallRandomBuilder())
```

Comment:
I intended this single line for a separate PR but added by accident. No problem all it does is speed up the test by creating a small model